packages/docsearch-react/src/DocSearchButton.tsx

```diff
@@ -34,14 +34,17 @@ export const DocSearchButton = React.forwardRef<HTMLButtonElement, DocSearchButt
     const [actionKeyReactsTo, actionKeyAltText, actionKeyChild] =
       key === ACTION_KEY_DEFAULT
         ? // eslint-disable-next-line react/jsx-key -- false flag
-          ([ACTION_KEY_DEFAULT, 'Ctrl', <ControlKeyIcon />] as const)
-        : (['Meta', 'Command', key] as const);
+          ([ACTION_KEY_DEFAULT, 'Control', <ControlKeyIcon />] as const)
```

Comment:
do we need this change? `Ctrl` -> `Control` ?
and `'Meta', 'Command', key` -> `'Meta', 'Meta', key`

Reply:
Yes, `Ctrl` and `Command` aren't valid values for `aria-keyshortcut`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts

`Control` and `Meta` are.